pages/jdl/applications.md

```diff
@@ -260,6 +260,18 @@ Here are the application options supported in the JDL:
     <td></td>
     <td></td>
   </tr>
+  <tr>
+    <td>blueprint</td>
+    <td></td>
+    <td>name of an additional blueprint (see <a href="https://www.jhipster.tech/modules/marketplace/#/list">Marketplace</a>)</td>
```

Comment:
Good idea to include the list!
Can you just capitalize "name", for consistency? thanks!